docs/modules/ROOT/pages/node_pattern.adoc

```diff
@@ -135,6 +136,22 @@ value:
 * `(int _ _)` will not match because `int` types have just one child that
 contains the value.
 
+You can specify a name name to make them more descriptive:
+
+----
+(send nil? $_method_name)
```

Comment:
Probably meant to not capture in the example here

```suggestion
(send nil? _method_name)
```

Reply:
Oops, yes. A bit early in the doc for making references to that